cloud/deployment/deployment.go

```diff
@@ -29,14 +29,14 @@ import (
 )
 
 var (
-	errInvalidDeployment      = errors.New("the Deployment specified was not found in this workspace. Your account or API Key may not have access to the deployment specified")
-	ErrInvalidDeploymentKey   = errors.New("invalid Deployment selected")
-	ErrInvalidClusterKey      = errors.New("invalid Cluster selected")
-	ErrInvalidRegionKey       = errors.New("invalid Region selected")
-	errTimedOut               = errors.New("timed out waiting for the deployment to become healthy")
-	ErrWrongEnforceInput      = errors.New("the input to the `--enforce-cicd` flag")
-	ErrNoDeploymentExists     = errors.New("no deployment was found in this workspace")
-	ErrInvalidResourceRequest = errors.New("invaild resource request")
+	errInvalidDeployment         = errors.New("the Deployment specified was not found in this workspace. Your account or API Key may not have access to the deployment specified")
+	ErrInvalidDeploymentKey      = errors.New("invalid Deployment selected")
+	ErrInvalidClusterKey         = errors.New("invalid Cluster selected")
+	ErrInvalidRegionKey          = errors.New("invalid Region selected")
+	errTimedOut                  = errors.New("timed out waiting for the deployment to become healthy")
+	ErrWrongEnforceInput         = errors.New("the input to the `--enforce-cicd` flag")
+	ErrInvalidResourceRequest    = errors.New("invaild resource request")
+	ErrNotADevelopmentDeployment = errors.New("the Deployment specified does not have development mode set")
```

Comment:
should we change to "the Deployment specified is not a development Deployment"? The error message as written makes it sounds like development mode is something that can be changed

Reply:
Happy to change that. Thanks, David.

Reply:
Done.